RFC/src/RFC-0301_NamespaceRegistration.md

```diff
@@ -133,20 +133,40 @@ linked to a [digital asset] on the DAN.
 
 **Req** - The OpenAlias TXT DNS record implementation MUST make provision to interpret entries that are made up of more than one string as defined in [[2]].
 
+**Req** - The OpenAlias TXT DNS record MUST adhere to the formatting requirements as specified in [[1]].
+
 **Req** - The OpenAlias TXT DNS record MUST be constructed as follows:
 
 | OpenAlias TXT DNS Record Field | OpenAlias TXT DNS Record Data                                |
```

Comment:
Have we looked at the size limitations of a TXT field? We have to keep that in consideration

Reply:
Covered in comment above.

Reply:
Yes, discussed in the introductory text under **OpenAlias TXT DNS Records** just before the requirement